eng/pipelines/coreclr/templates/xplat-pipeline-job.yml

```diff
@@ -103,6 +104,18 @@ jobs:
         - name: priorityArg
           value: '-priority=1'
 
+    - coreFxArtifactName: ''
```

Comment:
coreFx -> libraries here as well. We might just want to do it for all the yml files.